deploy/webhook-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
@sergioifg94 Will this automatically be included in a new CSV generation since it is under the deploy folder?